DungeonGame.py

```diff
@@ -18,17 +18,20 @@
 die = ("die", "Die", "I die", "I Die", "i die", "i Die", "dead", "Dead")
 inventory = ("Inventory", "inventory", "I", "i")
 quit = ("Quit", "quit", "Q", "q", "Exit", "exit", "E", "e")
+game_state = {}
```

Comment:
If `game_state` is going to be accessible by everything, we need to either
- pass it along from function to function
- declare it outside of the scope, and then within any function that needs it:
    ```
    global game_state  # to tell the function to use the global name, not a local one
    # now you can modify it from within the function
    ```